package.json

```
@@ -15,6 +15,7 @@
     "clean": "yarn clean:packages && yarn clean:web && rm -rf node_modules coverage tsconfig.tsbuildinfo",
     "clean:web": "rm -rf build tsconfig.web.tsbuildinfo",
     "clean:packages": "yarn workspaces foreach -ptiv --exclude @shapeshiftoss/web run clean && rm -rf tsconfig.packages.tsbuildinfo",
+    "codemod:clear-assets-migration": "jscodeshift -t src/state/migrations/transformClearAssets.ts --parser=tsx --extensions=ts,tsx src/state/migrations/index.ts",
```

can be more explicit given the file being transformed
```suggestion
    "codemod:clear-assets-migration": "jscodeshift -t src/state/migrations/transformClearAssets.ts --parser=ts --extensions=ts src/state/migrations/index.ts",
```

---

c3db66ed7cb0e07071830dfc66f95a23b86fffd7

---

And confirmed the ts parser is happy as well: a4e21cb7e27db6c1d589ea07c9b2d5a14bebf056